### PR TITLE
Added the documentation of lie algebras module using sphinx fixed Issue #9171

### PR DESCRIPTION
**Pull request by Chitrank-Dixit**
Added and generated the sphinx documentation for the Lie Algebra module.

### CODE REVIEW

File: doc/src/modules/liealgebras/index.rst [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+Dynkin Diagram
+=============
+.. autofunction:: sympy.liealgebras.dynkin_diagram.DynkinDiagram
+   
+   
+   
+
+
+
```
Comment (Anmolks) [inline doc/src/modules/liealgebras/index.rst]: This is not important but you can remove these newlines.

**Comment by Chitrank-Dixit** [comment]
@Anmolks now does it looks alright ?

**Comment by debugger22** [comment]
@Chitrank-Dixit Can you build the docs and show?

**Comment by Chitrank-Dixit** [comment]
@debugger22 Yes I have added the content in index.rst and then build the docs , using 
$ make html 

inside sympy/doc

please have a look at this generated documentation repository at github 
https://github.com/Chitrank-Dixit/sympy-docs/blob/master/_build/html/modules/liealgebras/index.html

**Comment by debugger22** [comment]
You'll also have to make changes to this file `https://github.com/sympy/sympy/blob/master/doc/src/modules/index.rst`.

Add following to that file:

```
liealgebras/index.rst
```

**Comment by debugger22** [comment]
Since you are at this, can you also fix a typo here
https://github.com/sympy/sympy/blob/master/sympy/liealgebras/type_a.py#L38
`with a 1 iin the ith position and a -1` -> `with a 1 in the ith position and a -1`

**Comment by debugger22** [comment]
Also give two newlines here https://github.com/sympy/sympy/blob/master/sympy/liealgebras/type_a.py#L25

Before and after
```
Example
=======
```

**Comment by debugger22** [comment]
These suggestions are for all types(type_a.py, type_b.py and so on).

**Comment by debugger22** [comment]
Fix the example in `simple_root` too. Again for all types.

**Comment by debugger22** [comment]
Fix the example in `sympy.liealgebras.cartan_matrix.CartanMatrix(ct)`

**Comment by debugger22** [comment]
@Chitrank-Dixit Let's finish this.

**Comment by Chitrank-Dixit** [comment]
Sorry @debugger22 I was busy with my College project It was the last project of my college I am now a Master of Computer Application. I am looking for this issue and fix it soon. thanks for the valuable feedbacks.

**Comment by czgdp1807** [comment]
I think the documentation for lie algebras is already there in `doc/src/modules/liealgebras/index.rst` and [here](https://docs.sympy.org/latest/modules/liealgebras/index.html). Probably it can be closed.

**Comment by czgdp1807** [comment]
Closing this in favour of https://docs.sympy.org/latest/modules/liealgebras/index.html
Thanks for the contributions.

**Pull request closed without merging**